tests/fixtures/config/hooks/plugin_with_ui.py

```diff
@@ -214,7 +214,7 @@ class PluginWithUi(HookBaseClass):
     Plugin for creating generic publishes in Shotgun
     """
 
-    def create_settings_widget(self, parent):
+    def create_settings_widget(self, parent, items):
```

Comment:
I'm not actually sure where this fixture is used?
Is this for the integration tests?

Reply:
Yes